db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/utilMain.java

```diff
@@ -845,4 +846,8 @@ int getCurrentRowNumber(ResultSet rs)
 	public final Object run() {
 		return  getClass().getResourceAsStream(ProductGenusNames.TOOLS_INFO);
 	}
+
+	public void setPromptClock(boolean showPromptClock) {
```

Comment:
Shouldn't that be static? Setting a static field from a nonstatic method is weird.

Reply:
Fixed.